pkg/discovery/worker/compliance/affinity_helper.go

```diff
@@ -0,0 +1,302 @@
+package compliance
+
+import (
+	"errors"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	api "k8s.io/client-go/pkg/api/v1"
+
+	"github.com/turbonomic/kubeturbo/pkg/discovery/util"
+
+	"github.com/golang/glog"
+)
+
+// NOTE: implementations are copied and modified from scheduler/predicate
```

Comment:
Change the comment to "implementation refers to the implementation of scheduler/predicate"